CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Changed
+
+- Pin jsonschema version to <4.18 until regresssions are fixed
```

Comment:
Is #1189 not enough?

Reply:
If you installed `test` but not `validation`, you end up with a version that doesn't work.

Reply:
because the test restriction is `~=4.17` instead of `~=4.17.0`, so any 4.18.x is valid to install for that constraint

Reply:
IMO we should use the same bounds introduced in #1189

Reply:
that's probably better.

Reply:
done